### ISSUE TITLE
[Bug] Logs displaying there was a issue with loot-table (loot table bountiful:blocks/bountyboard)

### ISSUE THREAD

**Issue by 3e2j** [ejektaflex/Bountiful#211]
This doesn't cause a crash - as far as I know - 

```
[12:09:45] [Render thread/ERROR]: Couldn't parse loot table bountiful:blocks/bountyboard
com.google.gson.JsonSyntaxException: Missing type, expected to find a string
	at Not Enough Crashes deobfuscated stack trace.(1.18+build.1) ~[?:?]
	at net.minecraft.util.JsonHelper.getString(JsonHelper:112) ~[?:?]
	at net.minecraft.loot.function.SetContentsLootFunction$Serializer.fromJson(SetContentsLootFunction:112) ~[?:?]
	at net.minecraft.loot.function.SetContentsLootFunction$Serializer.fromJson(SetContentsLootFunction:101) ~[?:?]
	at net.minecraft.loot.function.ConditionalLootFunction$Serializer.fromJson(ConditionalLootFunction:99) ~[?:?]
	at net.minecraft.loot.function.ConditionalLootFunction$Serializer.fromJson(ConditionalLootFunction:88) ~[?:?]
	at net.minecraft.util.JsonSerializing$GsonSerializer.deserialize(JsonSerializing:102) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:932) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1003) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162) ~[gson-2.8.8.jar:?]
	at net.minecraft.util.JsonHelper.deserialize(JsonHelper:448) ~[?:?]
	at net.minecraft.util.JsonHelper.deserialize(JsonHelper:466) ~[?:?]
	at net.minecraft.loot.entry.LeafEntry$Serializer.fromJson(LeafEntry:149) ~[?:?]
	at net.minecraft.loot.entry.LeafEntry$Serializer.fromJson(LeafEntry:129) ~[?:?]
	at net.minecraft.loot.entry.LootPoolEntry$Serializer.fromJson(LootPoolEntry:86) ~[?:?]
	at net.minecraft.loot.entry.LootPoolEntry$Serializer.fromJson(LootPoolEntry:74) ~[?:?]
	at net.minecraft.util.JsonSerializing$GsonSerializer.deserialize(JsonSerializing:102) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:932) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1003) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162) ~[gson-2.8.8.jar:?]
	at net.minecraft.util.JsonHelper.deserialize(JsonHelper:448) ~[?:?]
	at net.minecraft.util.JsonHelper.deserialize(JsonHelper:456) ~[?:?]
	at net.minecraft.loot.LootPool$Serializer.deserialize(LootPool:173) ~[?:?]
	at net.minecraft.loot.LootPool$Serializer.deserialize(LootPool:169) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:932) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1003) ~[gson-2.8.8.jar:?]
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162) ~[gson-2.8.8.jar:?]
	at net.minecraft.util.JsonHelper.deserialize(JsonHelper:448) ~[?:?]
	at net.minecraft.util.JsonHelper.deserialize(JsonHelper:466) ~[?:?]
	at net.minecraft.loot.LootTable$Serializer.deserialize(LootTable:213) ~[?:?]
	at net.minecraft.loot.LootTable$Serializer.deserialize(LootTable:209) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:932) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1003) ~[gson-2.8.8.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:975) ~[gson-2.8.8.jar:?]
	at net.minecraft.loot.LootManager.method_20711(LootManager:44) ~[?:?]
	at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
	at net.minecraft.loot.LootManager.apply(LootManager:42) ~[?:?]
	at net.minecraft.loot.LootManager.apply(LootManager:17) ~[?:?]
	at net.minecraft.resource.SinglePreparationResourceReloader.method_18790(SinglePreparationResourceReloader:13) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at net.minecraft.resource.SimpleResourceReload.method_18365(SimpleResourceReload:71) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor:151) ~[?:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor:23) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor:125) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor:134) ~[?:?]
	at net.minecraft.client.MinecraftClient.createIntegratedResourceManager(MinecraftClient:2154) ~[?:?]
	at net.minecraft.client.MinecraftClient.startIntegratedServer(MinecraftClient:2003) ~[?:?]
	at net.minecraft.client.MinecraftClient.startIntegratedServer(MinecraftClient:1943) ~[?:?]
	at net.minecraft.client.gui.screen.world.WorldListWidget$Entry.start(WorldListWidget:436) ~[?:?]
	at net.minecraft.client.gui.screen.world.WorldListWidget$Entry.play(WorldListWidget:342) ~[?:?]
	at net.minecraft.client.gui.screen.world.WorldListWidget$Entry.mouseClicked(WorldListWidget:278) ~[?:?]
	at net.minecraft.client.gui.widget.EntryListWidget.mouseClicked(EntryListWidget:359) ~[?:?]
	at net.minecraft.client.gui.ParentElement.mouseClicked(ParentElement:27) ~[?:?]
	at net.minecraft.client.Mouse.method_1611(Mouse:94) ~[?:?]
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen:491) ~[?:?]
	at net.minecraft.client.Mouse.onMouseButton(Mouse:94) ~[?:?]
	at net.minecraft.client.Mouse.method_22686(Mouse:165) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor:101) ~[?:?]
	at net.minecraft.client.Mouse.method_22684(Mouse:165) ~[?:?]
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36) ~[lwjgl-glfw-3.2.2.jar:build 10]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.2.2.jar:build 10]
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3174) ~[lwjgl-glfw-3.2.2.jar:build 10]
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem:218) ~[intermediary-1.18.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:1145) ~[?:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:733) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:236) [intermediary-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.8.jar:?]
```

**Comment by ejektaflex** [comment]
Yeah - this is purely cosmetic, but I will investigate. Thank you.

**Comment by romgerman** [comment]
I have that error too.

**Comment by nmurphy101** [comment]
Same

**Comment by ejektaflex** [comment]
Fixed in 6.0.3-1.20.1
